cmd/main.go

```diff
@@ -88,30 +91,38 @@ func main() {
 	}
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
+		Scheme: scheme,
+		Metrics: metricsserver.Options{
+			BindAddress: "0",
+		},
+		WebhookServer: &webhook.DefaultServer{
```

Comment:
We don't have a webhook, so wondering whether it's necessary to specify this

Reply:
We confirmed that the old 'Port' field also corresponded with webhook. So decided to leave it as is (set it using the new field)